worlds/metroidprime/data/Tricks.py

```diff
@@ -0,0 +1,807 @@
+            TrickDifficulty.Easy,
+            self.logic.can_space_jump,
+        )
+        self.tower_chamber_no_space_jump: TrickInfo = TrickInfo(
+            "Tower Chamber No Space Jump",
+            "Reach the Tower Chamber without Space Jump by using a double bomb jump",
+            TrickDifficulty.Easy,
+            rule_func=lambda world, state: self.logic.can_bomb(world, state)
+            and self.logic.can_move_underwater(world, state),
+        )
+
+        self.ruined_nursery_no_bombs: TrickInfo = TrickInfo(
+            "Ruined Nursery No Bombs",
+            "Reach the Ruined Nursery Item by space jumping and morphing near the item",
+            TrickDifficulty.Hard,
+            lambda world, state: self.logic.can_space_jump(world, state)
+            and self.logic.can_morph_ball(world, state),
+        )
+
+        self.magma_pool_scan_dash: TrickInfo = TrickInfo(
+            "Cross Magma Pool Suitless",
+            "Cross magma pool using a scan dash on the crate items",
+            TrickDifficulty.Medium,
+            lambda world, state: self.logic.can_space_jump(world, state)
+            and self.logic.can_scan(world, state),
+        )
+        self.magma_pool_debris_jump: TrickInfo = TrickInfo(
+            "Cross Magma Pool With SJB and Gravity",
+            "Use the space jump boots to jump off debris to cross the pool",
+            TrickDifficulty.Easy,
+            lambda world, state: self.logic.can_space_jump(world, state)
+            and self.logic.can_move_underwater(world, state)
+            and self.logic.has_energy_tanks(world, state, 2),
+        )
+        self.magma_pool_item_debris_jump: TrickInfo = TrickInfo(
+            "Magma Pool Debris Jump",
+            "Use the space jump boots to jump off debris to cross the pool",
+            TrickDifficulty.Easy,
+            lambda world, state: self.logic.can_space_jump(world, state)
+            and self.logic.can_move_underwater(world, state)
+            and self.logic.has_energy_tanks(world, state, 2)
+            and self.logic.can_power_bomb(world, state),
+        )
+        self.magma_pool_item_scan_dash: TrickInfo = TrickInfo(
+            "Magma Pool Item No Grapple",
+            "Use the scan dash and a power bomb to get the item in the magma pool",
+            TrickDifficulty.Medium,
+            lambda world, state: self.logic.can_space_jump(world, state)
+            and self.logic.can_scan(world, state)
+            and self.logic.can_power_bomb(world, state),
+        )
+        self.magma_pool_item_infinite_speed: TrickInfo = TrickInfo(
+            "Magma Pool Item Infinite Speed",
+            "Use infinite speed to get the item in the magma pool. Stay in morph ball and navigate to a save station or elevator afterward",
+            TrickDifficulty.Medium,
+            rule_func=lambda world, state: self.logic.can_infinite_speed(world, state)
+            and self.logic.can_bomb(world, state),

```

Comment:
```suggestion
            and self.logic.can_bomb(world, state)
            and (
                self.logic.can_heat(world, state)
                or self.logic.has_energy_tanks(world, state, 8)
            ),

```

This trick doesn't check for heat resistance or any energy, meaning it could be in logic suitless without any energy, which would be impossible since IS takes at least 50 seconds to attain. There's no specific reason I picked 8 E-Tanks besides it being enough to get IS with a reasonably fast spinup so that can be tweaked. It might also make sense to just always expect heat resistance for this since IS can be finicky.

This also raises the general question of whether you want all other Magma Pool tricks to expect hellrunning the room, though the other methods are much faster than this one.